src/components/ButtonPanel.js

```diff
@@ -26,6 +28,7 @@ const ButtonPanel = () => (
                 key={Math.random()}
                 color={(id !== 4 || i !== 2) && i !== 3}
                 wide={name === '0'}
+                onClick={onClick}
```

Comment:
[Optional] change the name of the prop onClick to better suit it's purpose may be a better name would be handleClick as suggested above from the screenshot

Reply:
Done!